cclib/bridge/cclib2psi4.py

```diff
@@ -7,19 +7,21 @@
 
 """Bridge for using cclib data in Psi4 (http://www.psicode.org/)."""
 
+import numpy as np
+
 from cclib.parser.utils import find_package
 
 _found_psi4 = find_package("psi4")
 if _found_psi4:
     from psi4.core import Molecule
 
 
-def _check_psi4(found_psi4):
+def _check_psi4(found_psi4: bool) -> None:

```

Comment:
Do you think it's prudent to annotate internal functions? Something I normally don't do.

Reply:
This comment should have been a response here: https://github.com/cclib/cclib/pull/1179#issuecomment-1454916110:
> I always do, since they may be reused many times, the body won’t get
checked if the signature isn’t annotated, and I’ve been bitten by this
before.